src/Sulu/Bundle/LocationBundle/Resources/js/package.json

```diff
@@ -6,7 +6,9 @@
     "dependencies": {
         "classnames": "^2.2.5",
         "leaflet": "^1.5.1",
-        "react-leaflet": "^3.1.0"
+        "react-leaflet": ">=3.1.0 <3.2.0 || ^3.2.1",
```

Comment:
this is a workaround to fix the build because of https://github.com/PaulLeCam/react-leaflet/issues/879. we should revert the change when https://github.com/PaulLeCam/react-leaflet/issues/879 is fixed.